CKWorkspace.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -3,7 +3,7 @@
     "pins": [
       {
         "package": "FHIRModels",
-        "repositoryURL": "git@github.com:apple/FHIRModels.git",
+        "repositoryURL": "https://github.com/apple/FHIRModels.git",
```

Comment:
ssh wouldn't play nicely with SPM in the xcodebuild command on travis, so I switched to https instead. The rest of the changes to the project (except for .travis.yml) are due to this change

Reply:
I think it makes sense to make this change in the root file too!

Reply:
Updated.

I got these errors while running `swift test` from cl:

```
error: the library 'CareKitFHIR' requires macos 10.10, but depends on the product 'ModelsR4' which requires macos 10.13; consider changing the library 'CareKitFHIR' to require macos 10.13 or later, or the product 'ModelsR4' to require macos 10.10 or earlier.
error: the library 'CareKitFHIR' requires macos 10.10, but depends on the product 'ModelsDSTU2' which requires macos 10.13; consider changing the library 'CareKitFHIR' to require macos 10.13 or later, or the product 'ModelsDSTU2' to require macos 10.10 or earlier.
```

I didn't change the build because I'm not running SPM 5.3 and I'm not sure if automatically resolves this

Reply:
Hmm, it seems like SPM is trying to run the tests on macOS instead of iOS.